frontend/src/metabase/query_builder/components/filters/modals/BulkFilterList/BulkFilterList.tsx

```diff
@@ -47,6 +54,27 @@ const BulkFilterList = ({
     [options],
   );
 
+  if (!dimensions.length && !segments.length) {
```

Comment:
I really prefer the boolean expression and I think reads naturally is "does this have a length?" instead of making a comparison.  But if the other is more consistent with established patterns, that's fine by me.